.github/workflows/linked_issues.yml

```
@@ -0,0 +1,18 @@
+name: Check linked issues
+
+on:
+  pull_request_target:
+    types: [opened, edited, reopened, synchronize]
+
+jobs:
+  check_pull_requests:
+    runs-on: ubuntu-latest
+    name: Check linked issues
+    permissions:
+      issues: read
+      pull-requests: write
+    steps:
+      - uses: nearform-actions/github-action-check-linked-issues@v1
+        id: check-linked-issues
+        with:
+          custom-body-comment: "No linked issue!"
```

I'd prefer we [disable comments](https://github.com/nearform-actions/github-action-check-linked-issues#disabling-comments) to PRs saying that they don't have a linked issue. Seeing the name of the CI job that failed `Check linked issues` should be enough.

```suggestion
        with:
          comment: false
```

 If not though, lets also add another step that checks the `linked_issues_count` output from this step and just echos out something along the line of `This CI job failed because you did not link an issue. In the PR description, make sure there is a `closes #xx` where the `xx` is replaced by the issue number that this PR closes